### ISSUE TITLE
Google reCAPTCHA

### ISSUE THREAD

**Issue by mflipe** [Battleman/zoomdl#7]
For security reasons, Zoom limit the number of times you can type the wrong password, there are a reasonable number of attempts but if you get it wrong too many times it will prompt for Google reCAPTCHA.
`zoomdl` do not work with the **reCAPTCHA**.
It got in to this trouble with this setup:

```
OS: Kubuntu 20.04
Password "9F!+$N0."
```

**Problem 1:** 
`bash: !+: event not found`
**Solution:** `set H+ ` ([StackOverflow](https://stackoverflow.com/questions/26443880/bash-event-not-found))

**Problem 2:** 
`Unable to open url (URL). Are you sure there is no password, or have you entered it correctly?`
- running `printf "9F!+$N0."` returns  `9F!+.`
- running `printf "9F!+\$N0."` returns  `9F!+$N0.`

**Problem 3:** 
`Unable to open url (URL). Are you sure there is no password, or have you entered it correctly?`
**reCAPTCHA**

**Summing up:**
**Problem 1** is related to `bash` and with passwords with `!`
**Problem 2** is also related to `bash` and with passwords with `$`
**Problem 3** occurs after a number of wrong attempts, in this case because of **Problem 1 and 2**

Hope it can be of help.

Regards,
Marcos

**Comment by Battleman** [comment]
Hi
Thanks for the feedback. For problem 1 and 2, I guess the problem is on the user side. Bash will indeed try a lot of things with strings in double quotes. I think putting between single quotes (`'`) solves both problems. I can probably update the README to suggest single quotes. I don't want to print the password though, doesn't seem a good thing to do. Perhaps in some verbose mode...

For problem 3, indeed CAPTCHA are a problem for this kind of automation. The best solution would be to open a browser and prompt the captcha, then close and continue, but I'm not sure this is easily doable. A second solution would be to simple inform the user and "fail gracefully".

**Comment by mflipe** [comment]
As stated in Problem 2 the problem may be between the input password and the password received by zoomdl.
I don't see any problem on printing the password since I gave it as parameter, also would prefer it in a `-v/--verbose`

I forgot to tell you that my purpose here is only informative, thanks to your application I got all the recordings. Thank you.

**Comment by Battleman** [comment]
OK, I will soon try to update the code with the proposed changes.
Cheers

**Comment by Battleman** [comment]
New version 2020.07.07 added a log-level flag (`-v`). Inputing `-v 0` will provide "debug" log level. Problem 3 will be tracked in #15 , so I'm closing this one.
